### DISCUSSION TITLE
Add back option to discard current change request

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#688] [0. General bugs and questions]
This used to be there before new updates from last week or so. It's not possible now to revert to the previous change state without archiving the current one. Simply provide an option to discard the current CR if needed.

**Comment by valentin0h** [comment]
Hey @ahcsco, we've made a few changes in the past weeks, both on the header of the space and change requests but also in the options we provide in the list of change requests. 

Could you provide the steps to reproduce the issue so we can check on our end?

**Reply by ahcsco** [reply]
Yes, it seems to be connected to recent changes.
There used to be a back arrow button with the commit message to revert to the previous state.
Now, you need to click around on other versions or archive it to revert.
So if you create a new change request and then try to discard the CR without merging/reviewing changes, you'll see what I mean.
What I'm looking at is a discard changes button or something like that.
